pkg/console/subresource/configmap/configmap.go

```diff
@@ -27,6 +27,13 @@ func getApiUrl(infrastructureConfig *configv1.Infrastructure) string {
 	return ""
 }
 
+func getCliDownloadUrl(downloadsRoute *routev1.Route) string {
+	if downloadsRoute != nil {
+		return downloadsRoute.Spec.Host
```

Comment:
`route.Spec.Host` will actually be removed, we have a `GetCanonicalHost()` func that could be used instead.